evaluators/rank/NdcgEvaluator/tests/test_ndcg_evaluator.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from .. import NDCGEvaluator
+
+@pytest.fixture(scope='function')
+def evaluator():
+    return NDCGEvaluator(eval_at=3)
+
+@pytest.fixture(scope='function')
+def desired():
+    return [.8, .4, .1]
+
+@pytest.mark.parametrize('actual, expected',[
```

Comment:
make sure to test edge cases, where some of them are empty and so on...

Reply:
done